### PR TITLE
fix(init): refactored the init.ts success message

### PR DESCRIPTION
**Pull request by pranshuchittora**
Previously there was multiple `stdout.write` due to the presence of conditional statements. In this PR I have scrapped those repeating lines into a single `process.stdout.write`

**What kind of change does this PR introduce?**
Refactoring

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
Can't say

**Does this PR introduce a breaking change?**
Probably not

### CODE REVIEW

File: packages/init/init.ts [hunk 1]
```diff
@@ -77,22 +77,14 @@ export default function runTransform(webpackProperties: IWebpackProperties, acti
...
+		successMessage = `Congratulations! ${webpackProperties.config.item} has been ${action}ed!`;
+
 	} else {
-		process.stdout.write(
-			"\n" +
-				chalk.green(
-					"Congratulations! Your new webpack configuration file has been created!\n",
-				),
-		);
+		successMessage = "Congratulations! Your new webpack configuration file has been created!";
```
Comment (dhruvdutt) [inline packages/init/init.ts]: Can you keep this as the default value of `successMessage` and clean up the `else` block?

Comment (pranshuchittora) [reply packages/init/init.ts]: Yup makes sense, thanks for the lightning fast review :zap: .

File: packages/init/init.ts [hunk 2]
```diff
@@ -78,21 +78,10 @@ export default function runTransform(webpackProperties: IWebpackProperties, acti
...
-		process.stdout.write(
-			"\n" +
-				chalk.green(
-					"Congratulations! Your new webpack configuration file has been created!\n",
-				),
-		);
+		successMessage = `Congratulations! ${webpackProperties.config.item} has been ${action}ed!`;
+
 	}
+	process.stdout.write("\n" + chalk.green(`${successMessage}\n`));
```
Comment (rishabh3112) [inline packages/init/init.ts]: Refactor it to template literal would be great.

Comment (pranshuchittora) [reply packages/init/init.ts]: Ya it can be done but it will reduce the readability & cleanliness, as `` nested inside `` .....

**Comment by pranshuchittora** [comment]
@evenstensberg screenshot of changes ?

**Comment by evenstensberg** [comment]
Take a screenshot of the change

**Comment by pranshuchittora** [comment]
@evenstensberg  screenshot captured :+1:

**Comment by evenstensberg** [comment]
The visual change, not code

**Comment by pranshuchittora** [comment]
No change as such.
In order to check for visual differences, I have added :tada:  in the string for testing purposes.

**Before**
Screenshot from 2019-04-01 19-20-51

**After**
Screenshot from 2019-04-01 19-21-10
_(The emoji is just for testing, its not been included in the PR)_

Hence,this PR doesn't introduces and visual change.

**Pull request merged**